sapi/fpm/php-fpm.8.in

```diff
@@ -175,6 +175,12 @@ The PHP-FPM CGI daemon will work well with most popular webservers, including Ap
 .PD 1
 .P
 .SH SEE ALSO
+For a more or less complete description of PHP-FPM look here:
+.PD 0
+.P
+.B https://www.php.net/fpm
+.PD 1
+.P
```

Comment:
This hsould really replace the link to http://www.php.net/manual/  below and not adding another one. It would be also good to delete the link to php-fpm.org when you are in it as it's very outdated and not maintained.

Reply:
Good idea. Removed in the next commit. Thanks for the review.